src/components/Error.svelte

```diff
@@ -8,7 +8,7 @@ $: e = $error
   <div class="alert alert-danger alert-dismissible" role="alert">
     <h4 class="alert-heading">Something went wrong</h4>
     
-    <p class="text-monospace">{@html e.message || 'No additional details'}</p>
+    <p class="text-monospace">{@html e.message}</p>
```

Comment:
Not related to this change, but what's the use case for supporting HTML content in the error messages returned by the API?